lucid/utils.py

```diff
@@ -203,3 +208,17 @@ def check_stale_cache():
         _HAPPI_CACHE = (time.monotonic(), list(client.search(as_dict=True)))
 
     return _HAPPI_CACHE[1]
+
+
+@contextlib.contextmanager
+def no_device_lazy_load():
+    '''
+    Context manager which disables the ophyd.device.Device
+    `lazy_wait_for_connection` behavior and later restore its value.
+    '''
+    old_val = Device.lazy_wait_for_connection
+    Device.lazy_wait_for_connection = False
```

Comment:
I think it's more semantically clear if you put the `= False` setting is within the `try` block.
It's saying "do this; but after make sure to undo it". Not necessary to change though 🤷‍♂

Reply:
Addressed via https://github.com/pcdshub/lucid/pull/25/commits/286445199807482f14fb98ab20c4eccd9b2f2c83